_sources/Unit2-Using-Objects/topic-2-1-objects-intro-turtles.rst

```diff
@@ -44,14 +44,39 @@
 Objects - Instances of Classes
 ===============================
 
+Java is an **object-oriented programming** language. That means that after we’ve
+exhausted the possibilities of primitive types like ``int``, ``double``, and
+``boolean``, one of the primary ways of organizing our programs is in terms of
```

Comment:
Not sure about this sentence  and its two not so related parts: "That means that after we’ve
exhausted the possibilities of primitive types like ``int``, ``double``, and
``boolean``, one of the primary ways of organizing our programs is in terms of
**objects**."